DQM/DTMonitorModule/python/dtDQMOfflineSources_Cosmics_cff.py

```diff
@@ -42,11 +46,12 @@
 from DQM.DTMonitorModule.dtTriggerEfficiencyTask_cfi import *
 
 dtSourcesCosmics = cms.Sequence(dtDataIntegrityUnpacker  +
-                                DTDataIntegrityTask +
-                                dtDCSByLumiMonitor + 
-                                dtRunConditionVar + 
-                                dtSegmentAnalysisMonitor +
-                                dtResolutionAnalysisMonitor +
-                                dtEfficiencyMonitor +
-                                dtTriggerEfficiencyMonitor +
-                                dqmInfoDT)
+                         twinMuxStage2Digis +
```

Comment:
Hi @davidlange6 , @rekovic , @dmitrijus , could any help me to understand where the unpacker should go instead? 
For DQM online, it is sourced through DQM/DTMonitorModule/python/dt_dqm_sourceclient_common_cff.py in DQM/Integration/python/clients/dt_dqm_sourceclient-live_cfg.py 
But the equivalent for DQM offline seems to be DQM/DTMonitorModule/python/dtDQMOfflineSources_*.py
Any hint is very apreciated
Thank you very much